routes/web.php

```diff
@@ -42,14 +42,20 @@
     //route for announcements
     Route::get('/dashboard/announcements', [HubController::class, 'getAnnouncements'])->name('announcements');
 
-    //route for my profile in personal hun
+    //route for my profile in personal hub
     Route::get('/dashboard/profile', [HubController::class, 'getProfileInfo'])->name('my-profile');
 
     //route for personal programme
     Route::get('/dashboard/programme', [HubController::class, 'getProgramme'])->name('my-programme');
 
+    //route for page for registering for presentations
+    Route::get('/dashboard/programme/register', [HubController::class, 'registerForPresentations'])->name('my-programme-register');
+
+    //route for enrolling for a presentation
+    Route::get('/dashboard/programme/register/attach/{presentationId}', [HubController::class, 'attachParticipation'])->name('create-participant');
```

Comment:
Since it alters data it may be a better idea to be a POST or a PUT - security wise.